client/src/components/Projects/ColumnHeaderPopups/VisibilityPopup.js

```diff
@@ -41,6 +49,9 @@ const VisibilityPopup = ({
     close();
   };
 
+  const handleChangeVisibility = visibilityValue => {
```

Comment:
Don't want to apply the change to the filter when the user picks a choice from the list - need to wait until they click on the Apply button.